data/data/agent/files/usr/local/bin/issue_status.sh

```diff
@@ -1,25 +1,19 @@
 #!/bin/bash
 
 issue_file() {
-    printf "/etc/issue.d/%s.issue" "$1"
+    printf "/etc/issue.d/%s" "$1"
 }
 
 set_issue() {
     local outfile
     outfile="$(issue_file "$1")"
-    local tmp
-    tmp="$(mktemp)"
```

Comment:
One advantage of using mktemp is that there is no way for two processes running this code to interfere with each other. I think we could continue to get the benefits by doing

```
tmp=$(mktemp -p /etc/issue.d/)
```

and then none of the other code needs to change.